cmd/example-mysql/main.go

```diff
@@ -242,3 +245,22 @@ func parseTileIndexWidth(index string) (uint64, uint64, error) {
 
 	return i, w, nil
 }
+
+func initDatabaseSchema(ctx context.Context, db *sql.DB) {
+	if db != nil && *initSchemaPath != "" {
+		rawSchema, err := os.ReadFile(*initSchemaPath)
+		if err != nil {
+			klog.Exitf("Failed to read init schema file %q: %v", *initSchemaPath, err)
+		}
+
+		queries := strings.SplitAfter(strings.TrimSpace(string(rawSchema)), ";")
```

Comment:
This makes me uncomfortable as it isn't escaping aware etc. Could we use https://github.com/go-sql-driver/mysql?tab=readme-ov-file#multistatements and throw the whole file at it?

Reply:
That's a great suggestion. Done, thanks!